JASP-Engine/JASP/po/R-nl.po

```diff
@@ -6337,73 +6756,312 @@ msgstr "Posterior Kengetallen Voor Submodel %s"
+msgstr "De analyse is onderbroken met de volgende foutmelding:"
+
+msgid "Frequentist Scale Reliability Statistics"
+msgstr "Frequentistische Schaal Validiteitsstatistieken"
+
+msgid "Point estimate"
+msgstr "Puntschatting."
 
 msgid ""
-"%s\n"
-"Warning: Greatest lower bound can only be calculated for three or more "
-"variables."
+"%sMcDonald's ω estimation method switched to PFA because the CFA\n"
+"                                          did not find a solution."
 msgstr ""
-"%s\n"
-"Waarschuwing: Grootste ondergrens kan alleen worden berekend voor drie of "
-"meervariabelen."
+"%sMcDonald's ω schattingsmethode is omgeschakeld naar PFA omdat de CFA\n"
+"                                          geen oplossing vond."
 
 msgid "If item dropped"
 msgstr "Als item wordt weggelaten"
 
-msgid "item-rest correlation"
-msgstr "item-rest correlatie"
+msgid "Frequentist Individual Item Reliability Statistics"
+msgstr "Frequentistische Individuele Item Validiteitsstatistieken"
+
+msgid ""
+"McDonald's ω estimation method for item-dropped statistics\n"
+"                                    switched to PFA because the CFA did not "
+"find a solution."
+msgstr ""
+"%sMcDonald's ω schattingsmethode is omgeschakeld naar PFA omdat de CFA\n"
+"                                          geen oplossing vond."
+
+msgid "Fit Measures of Single Factor Model Fit"
+msgstr "Pasmaat van Enkele Factor Model passing"
+
+msgid ""
+"Fit measures cannot be displayed because the McDonald's ω \n"
+"                                               estimation method switched \n"
+"                                               to PFA as the CFA did not "
+"find a solution."
+msgstr ""
+"De passingmaat kan niet worden getoond omdat de schattingsmethode voor McDonald's ω \n"
+"veranderd is naar PFA, want de CFA leidde niet tot een oplossing."
+
+msgid "The priors for publication bias were set incorrectly."
+msgstr "De a priori verdeling voor publicatieonzuiverheid zijn verkeerd ingesteld."
+
+msgid "MCMC error"
+msgstr "MCMC fout"
+
+msgid "ESS"
+msgstr "ESS"
+
+msgid "%s is on %s scale."
+msgstr "s is op %s schaal."
+
+msgid "<em>p</em>-values interval %s"
+msgstr "<em>p</em>-waarden interval %s"
+
+msgid "The weights (%s) correspond to %s <em>p</em>-values."
+msgstr "De gewichten (%s) corresponderen met de %s <em>p</em>-waarden."
+
+msgid "Estimated studies' effects (%s) correspond to effect size %s."
+msgstr ""
+
+msgid "Effect size (%s)"
+msgstr "Effectgrootte (%s)"
+
+msgid "Heterogeneity (%s)"
+msgstr "Heterogeniteit (%s)"
+
+msgid "Prior Plots"
+msgstr "Prior Grafieken"
+
+msgid "Weight Function"
+msgstr "Gewichtsfunctie"
+
+msgid "Model Preview"
+msgstr "Model Voorvertoning"
+
+msgid ""
+"At least one prior distribution per parameter must be specified (either null "
+"or alternative)."
+msgstr ""
+"Tenminste een a priori distributie per parameter moet gespecificeerd zijn "
+"(in het null of alternatief model)."
```

Comment:
"Er moet tenminste één a priori verdeling gespecificeerd zijn per parameter "
"(in het null of alternatieve model)."